04/dbernhard/simple_while.oat

```diff
@@ -0,0 +1,11 @@
+
+
```

Comment:
Can you trailing remove whitespaces? Also please add a newline at the end of the file